cascade/src/main/java/me/saket/cascade/CascadePopupMenu.kt

```diff
@@ -69,6 +75,31 @@ open class CascadePopupMenu @JvmOverloads constructor(
       popup.contentView.background = it
     }
 
+    styler.overlayColor()?.let {
+      check(context is Activity) { "Activity context is required in order to add an overlay view" }
+      val container = (context.window.decorView as ViewGroup)
+      val overlay = View(context, null, 0).apply {
+        alpha = 0f
+        setBackgroundColor(it)
+      }
+      container.addView(overlay)
+
+      val alphaInAnimator = ObjectAnimator.ofFloat(overlay, View.ALPHA, 1f).apply {
+        duration = 500
```

Comment:
It's important to sync these two animations here `PopupWindow`'s transition. See `popup.enterTransition` and `popup.exitTransition` and reuse their duration and interpolator?

Reply:
`enterTransition` & `exitTransition` are only available from Nougat. Have added default fallback values for animation properties.